unity/Assets/Scripts/Util/Math/MathUtil.cs

```diff
@@ -58,7 +58,7 @@ public static bool EqualsEps(double a_val1, double a_val2, double eps = EPS)
         /// <param name="a_val1"></param>
         /// <param name="a_val2"></param>
         /// <returns></returns>
-        public static bool EqualsEps(Vector2 a_val1, Vector2 a_val2, double eps = EPS)
+        public static bool EqualsEps(this Vector2 a_val1, Vector2 a_val2, double eps = EPS)
```

Comment:
Perhaps keep both methods, one extension method and one normal method? Or does C# do that automatically?

Reply:
It can be used as both. This is some C# magic. Allows for easier calling. It can be called on an object of the parameter with this (thus Vector2) or just as a normal method.